zenith_utils/src/pq_proto.rs

```diff
@@ -164,7 +164,7 @@ impl FeStartupMessage {
             Err(e) => return Err(e.into()),
         };
 
-        if len < 4 || len > MAX_STARTUP_PACKET_LENGTH {
+        if !(4..=MAX_STARTUP_PACKET_LENGTH).contains(&len) {
```

Comment:
This is much less readable than the original IMHO

Reply:
What do you think is better, put an allow mark here, or ignore this lint globally?

Reply:
put an allow in `zenith_utils/lib.rs`